docs/_docs/Develop/script-js.md

```diff
@@ -258,6 +267,9 @@ If the script requires a special registry setting, there are 2 options:
 
 2. If the setting is special for this script, use a registry file. Create a `registry.reg` in `<scriptname>/resources` (see [IE6](https://github.com/PhoenicisOrg/scripts/blob/master/Applications/Internet/Internet%20Explorer%206.0/resources/ie6.reg)) and apply this in `pre/postInstall()` via
 ```javascript
+include(["utils", "functions", "apps", "resources"]);
+include(["engines", "wine", "plugins", "regedit"]);
+    ...
```

Comment:
Why the indention?

Reply:
I did it in the spirit of the indent in #### Pre/Post install hooks.